appbars/README.md

```diff
@@ -10,7 +10,11 @@ By inheriting the Vitamin theme in your app, you just have to use the default ma
 component or the Vitamin subclass component.
 
 ```kotlin
-implementation("com.decathlon.vitamin:appbars:<version>")
+implementation("com.decathlon.vitamin:vitamin:<version>")
+```
+
```

Comment:
Add a sentence to explain what is `Theme.Vitamin.x` and a link to the README in `vitamin` module.